src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -1657,6 +1670,7 @@ fd_exec_vm_syscall_test_run( fd_exec_instr_test_runner_t * runner,
   if( !fd_exec_test_instr_context_create( runner, ctx, input_instr_ctx, alloc, skip_extra_checks ) )
     goto error;
```

Comment:
I'm pretty sure we can just use spad here too (like in the valloc)

Reply:
Let's migrate these non critical path test harness allocations in a future PR, and keep this PR about live or replay transaction execution?